scripts/process_diagnostic_messages.rs

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env rust-script
```

Comment:
[`rust-script`](https://github.com/fornwall/rust-script) looked like a nice way for writing "scripts" in Rust